### PR TITLE
Prepare for production deployment

### PR DESCRIPTION
**Pull request by signalnerve**
Prep for production deployment by adding `wrangler.production.toml`, which will replace `wrangler.toml` at the end of workflows and allow deploying to the internal Cloudflare Docs account. When the docs go live, we can uncomment the `production` environment config option in the workflow and begin deploying to the final production URL.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -28,8 +28,16 @@ jobs:
...
       run: wrangler publish
       env:
         CF_API_KEY: ${{ secrets.CF_API_KEY }}
         CF_EMAIL: ${{ secrets.CF_EMAIL }}
+    - name: Copy wrangler.production.toml to wrangler.toml
+      run: cp wrangler.production.toml wrangler.toml
+    - name: Publish site
+      uses: cloudflare/wrangler-action@1.2.0
+      with:
+        apiToken: ${{ secrets.CF_PRODUCTION_API_TOKEN }}
```
Comment (signalnerve) [inline .github/workflows/deploy.yml:41]: @adamschwartz can i get admin access on this repo so i can manage this secret?

Comment (adamschwartz) [reply .github/workflows/deploy.yml:41]: Of course! You should have admin access now.

File: wrangler.production.toml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# wrangler.toml file for deploying to internal cloudflare zones/workers.dev instances.
+# this shouldn't be touched or used besides by the GitHub Actions workflow!
+
+name = "workers-docs"
+type = "webpack"
+account_id = "b54f07a6c269ecca2fa60f1ae4920c99" # cloudflaredocs.workers.dev
```
Comment (adamschwartz) [inline wrangler.production.toml:6]: Let’s make this its own env, something like `[env.migration-testing]` (or `[env.migration]`) and put that, plus the `[env.production]` stuff below, into the default `wrangler.toml`.

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -28,8 +28,16 @@ jobs:
...
         CF_API_KEY: ${{ secrets.CF_API_KEY }}
         CF_EMAIL: ${{ secrets.CF_EMAIL }}
-    - name: Upload assets
+    - name: Upload assets (development)
       run: wrangler publish
       env:
         CF_API_KEY: ${{ secrets.CF_API_KEY }}
         CF_EMAIL: ${{ secrets.CF_EMAIL }}
+    - name: Copy wrangler.production.toml to wrangler.toml
+      run: cp wrangler.production.toml wrangler.toml
```
Comment (adamschwartz) [inline .github/workflows/deploy.yml:37]: Then we can lose this as well.

File: wrangler.toml [hunk 4]
```diff
@@ -7,3 +7,14 @@ kv-namespaces = [{ binding = "REDIRECTS", id = "7df3fe779d0048d1bacc5d753c1153d0
 [site]
 bucket = "./public"
 entry-point = "workers-site"
+
+[env.migration]
+name = "workers-docs"
+account_id = "b54f07a6c269ecca2fa60f1ae4920c99" # cloudflaredocs.workers.dev
```
Comment (adamschwartz) [inline wrangler.toml:13]: Should say `cloudflare-docs.workers.dev` (with the hyphen), yea?

Comment (signalnerve) [inline wrangler.toml:13]:
```suggestion
account_id = "b54f07a6c269ecca2fa60f1ae4920c99" # cloudflare-docs.workers.dev
```

**Pull request merged**